test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -42,7 +42,7 @@ u256 exp256(u256 _base, u256 _exponent)
 	u256 result = 1;
 	while (_exponent)
 	{
-		if (static_cast<limb_type>(_exponent) & 1)	// If exponent is odd.
+		if (_exponent & 1)
```

Comment:
I'm not sure I understand why the cast was necessary.

If I understand correctly `limb_type` is typedef to a machine word (32/64 bit), and the intention in the unpatched version was to downcast u256 to a native word and logical AND it with 1 so that we test for even/odd.